src/library/Cardano/Fee.hs

```diff
@@ -331,26 +347,84 @@ coverRemainingFee (Fee fee) = go [] where
 -- >>> reduceChangeOutputs (DustThreshold 0) (Fee 15) (Coin <$> [1, 2, 4, 8])
 -- []
 --
-reduceChangeOutputs :: DustThreshold -> Fee -> [Coin] -> [Coin]
-reduceChangeOutputs threshold (Fee totalFee) changeOutputs
-    | totalFee >= totalChange =
-        []
-    | otherwise =
-        case positiveChangeOutputs of
-            x : xs -> x :| xs
-                & distributeFee (Fee totalFee)
-                & fmap payFee
-                & coalesceDust threshold
-            [] -> []
-  where
-    payFee :: (Fee, Coin) -> Coin
-    payFee (Fee f, c) =
-        fromMaybe C.zero (c `C.sub` f)
-
-    positiveChangeOutputs :: [Coin]
-    positiveChangeOutputs = filter (> C.zero) changeOutputs
-
-    totalChange = F.fold changeOutputs
+reduceChangeOutputs
+    :: FeeOptions i o
+    -> CoinSelection i o
+    -> Either (FeeAdjustmentError i o) (CoinSelection i o, Fee)
+reduceChangeOutputs opts s = do
+    -- The original requested fee amount
+    let Fee φ_original = estimateFee (feeEstimator opts) s
```

Comment:
Hi @KtorZ 

Perhaps something like `feeOriginal` would be clearer here?

Reasons:
- For people reading the code:
    - with the prefix "fee", the comment explaining that this is a fee becomes unnecessary. 
    - it's perhaps not obvious to all that the "φ" consonant maps to "f" .
- For maintainers:
    - Latin letters are (arguably) much easier to type.

Reply:
Marking this as resolved, as this has been discussed elsewhere. :+1: